assets/components/CalendarButton.jsx

```diff
@@ -28,8 +26,12 @@ class CalendarButton extends React.Component {
     render() {
         const isStartDateToday = moment.isMoment(this.state.startDate) && !this.state.startDate.isSame(moment(), 'day');
         const datePicker = (<DatePicker
-            customInput={<CalendarButtonWrapper active={isStartDateToday}/>}
-            dateFormat={AGENDA_DATE_PICKER_FORMAT_SHORT}
+            customInput={
+                <CalendarButtonWrapper
+                    active={isStartDateToday}
+                    date={moment(this.state.startDate).format('YYYY-MM-DD')}
```

Comment:
I don't see date prop in `CalendarButtonWrapper` so not sure what this does actually?

Reply:
my bad its typo. here we have to pass `value` prop